### PR TITLE
Common Headers

### PR DESCRIPTION
**Pull request by alefranz**
# Description of the Change

Add Common Headers documentation

## Value

Standardizing headers, in particular to add a session id standard concept.

### CODE REVIEW

File: standards/CommonHeaders.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## API-Version Header
+
+The `API-Version` header is used to indicate the version of the API being used. This header is important for managing backward compatibility and ensuring that clients can safely upgrade to newer versions of the API without breaking their existing integrations.
+See [Versioning.md] for more details on this header.
+
+## Client-Session-Id Header
+
+The `Client-Session-Id` header is used to track a client's session across multiple API requests. This unique identifier is essential for maintaining session consistency and is particularly useful in scenarios where stateful interactions are necessary. It enables the system to correlate requests to a specific user session, facilitating better analytics, security audits, and troubleshooting.
+Each client MUST generate a unique `Client-Session-Id` for every session, and consistently include it in the headers of all API requests made during that session.
+Each API MUST accept a `Client-Session-Id` header in every request, even when it is not needed for it's own processing, and pass it trough to any other API called.
```
Comment (dturveyza) [inline standards/CommonHeaders.md:24]: spelling of "through" is incorrect

**Comment by dturveyza** [comment]
Do we need to add SHOULD/MUST or REQUIRED/OPTIONAL to each header? The intro says "should be used", and then client session id says MUST.

File: standards/CommonHeaders.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+The `Content-Type` header specifies the media type of the request or response payload. It helps the server understand how to parse and process the data being sent or received. Common values for this header include `application/json`, `application/xml`, and `application/x-www-form-urlencoded`.
+
+## Accept Header
+
+The `Accept` header is used by the client to specify the media types it can handle in the response. It allows the server to send the response in the most appropriate format based on the client's preferences. This header is particularly useful when the API supports multiple response formats.
+
+## API-Version Header
+
+The `API-Version` header is used to indicate the version of the API being used. This header is important for managing backward compatibility and ensuring that clients can safely upgrade to newer versions of the API without breaking their existing integrations.
+See [Versioning.md] for more details on this header.
```
Comment (wordshaker) [inline standards/CommonHeaders.md:18]: Is this supposed to be a link?

**Comment by alefranz** [comment]
> Do we need to add SHOULD/MUST or REQUIRED/OPTIONAL to each header? The intro says "should be used", and then client session id says MUST.

Good point. Do you want to work on a separate PR @dturveyza once this is merged?

File: standards/CommonHeaders.md (hunk 2 above)
Comment (alefranz) [reply standards/CommonHeaders.md]: fixed it

**Review by wordshaker — APPROVED** [review]

**Pull request merged**